nixos/tests/restic.nix

```diff
@@ -137,6 +137,13 @@ import ./make-test-python.nix (
               pruneOpts = [ "--keep-last 1" ];
               checkOpts = [ "--some-check-option" ];
             };
+            customUnitConfig = {
+              inherit passwordFile paths;
+              repository = remoteRepository;
+              unitConfig = {
+                documentation = [ "custom-unit-config-was-written" ];
```

Comment:
I would add `serviceConfig = { something here };` to make sure to catch the issue with the `//` operator.